src/TimeSeries/widget/TimeSeries.js

```diff
@@ -220,10 +220,12 @@ define([
 
 
         _marshallSources: function (graphData) {
+          var maxValue = 0;
           var svgNode = this.svgNode;
           var data = [];
           var _widget = this;
           graphData.metrics.map(function(metric){
+            maxValue = Math.max( ...metric.values );

```

Comment:
move this line down to line above line 245 because this value is only relevant when graph type equals `line`